test/unit/user/user.spec.js

```diff
@@ -36,9 +53,8 @@ describe('Users Authentication', () => {
   });
 
   after(async () => {
-    await User.destroy({ cascade: true, truncate: true });
-    await Article.destroy({ cascade: true, truncate: true });
-
+    // await User.destroy({ cascade: true, truncate: true });
```

Comment:
Why is this code commented out?

Reply:
It was commented out while I was trying to debug my code with luqman.